src/outputgeneration/ParseTreeWriter.js

```diff
@@ -1006,9 +1006,12 @@ export class ParseTreeWriter extends ParseTreeVisitor {
    * @param {NewExpression} tree
    */
   visitNewExpression(tree) {
+    var complexOperand = tree.operand.operand !== void 0;
```

Comment:
We need to add a transformNewExpression to TemplateLiteralTransformer. Not yet sure how to best detect that the operand contains a template literal.